lib/ripper-tags/parser.rb

```diff
@@ -320,6 +324,8 @@ def on_assign(name, rhs, line, *junk)
         namespace = namespace + parts
       end
 
+      process(rhs)
```

Comment:
Why was this `process` added?

Reply:
If we don't add this `process`, the `Visitor` won't discover potential tags on the RHS of a constant assignment. If I remove this call my added test case fails with:
```
Failure: test_nested_constant_definitions(TagRipperTest)
/Users/owenstephens/code/ripper-tags/test/test_ripper_tags.rb:92:in `test_nested_constant_definitions'
     89:       }
     90:     EOC
     91:
  => 92:     assert_equal ('A'..'G').to_a, tags.map { |t| t[:name] }.sort
     93:     tags.each do |t|
     94:       assert_equal t[:name], t[:full_name]
     95:     end
<["A", "B", "C", "D", "E", "F", "G"]> expected but was
<["A", "D"]>
```
as the visitor is no longer looking "inside" the expressions in the (top-level) assignments

Reply:
Thanks for the explanation. Are we interested in processing RHS at all?

Reply:
I'm not sure I understand the question. 

We are interested in processing the RHS since we wouldn't otherwise discover the constant definitions on the RHS of the top-level assignment, which was the aim of this PR. Indeed, if we _don't_ process the RHS, in my original example we only get tags for `STATUSES` and `DISPLAY_MAPPING`; _with_ the `process(rhs)` call we also get tags for `OPEN` and `CANCELLED` as desired.